.github/workflows/test.yml

```diff
@@ -18,3 +18,27 @@ jobs:
 
       - name: Run tests
         run: npm run test
+  
+  end2end-tests:
+    runs-on: ubuntu-latest
+    
+    steps:      
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install
+        run: npm install
+
+      - name: Build static data
+        run: npm run build-mock-static-data
+
+      - name: Start mock server
+        run: npm run serve-website &
```

Comment:
is the & at the end needed?

Reply:
Yes to run it in the background and not to wait to complete.